external/external.sh

```diff
@@ -329,12 +330,17 @@ if [ $command_type == "load" ]; then
 
 			restore_docker_image_name_list=()
 
-			echo "Testing images from nightly builds"
-			image_os_micro_architecture_list=($criu_combo_os_microarch_list)
-			for image_os_micro_architecture in ${image_os_micro_architecture_list[@]}
-			do
-				restore_docker_image_name_list+=("${docker_registry_url}/$job_name/${JDK_VERSION}-${JDK_IMPL}-${docker_os}-${platform}-${image_os_micro_architecture}:${build_number}")
-			done
+			if [[ $job_name == "Grinder" ]]; then
```

Comment:
`job_name` is a local variable. To make this simple, I think we should use `$JOB_NAME` directly and remove line 31

Reply:
The PR has been updated, and issue related to PingPerf has been opened, which will resolve in another PR. Thanks.